docs/user/metal/install_ipi.md

```diff
@@ -0,0 +1,225 @@
+
+### Supported Hardware
+
+The architecture is intended to support a wide variety of hardware.  This was
+one of the reasons Ironic is used as an underlying technology.  However, so far
+development and testing has focused on PXE based provisioning using IPMI for
+out-of-band management of hosts.  Other provisioning approaches will be added,
+tested, and documented over time.
+
+## Installation Process
+
+Once an environment has been prepared according to the documented
+pre-requisites, the install process is the same as other IPI based platforms.
+
+`openshift-install create cluster`
+
+However, it is recommended to prepare an `install-config.yaml` file in advance,
+containing all of the details of the bare metal hosts to be provisioned.
+
+### Install Config
+
+The `install-config.yaml` file requires some additional details.  Most of the
+information is teaching the installer and the resulting cluster enough about
+the available hardware so that it is able to fully manage it.
+
+Here is an example `install-config.yaml` with the required `baremetal` platform
+details.
+
+**IMPORTANT NOTE:** The current install configuration for the `baremetal`
+platform should be considered experimental and still subject to change without
+backwards compatibility.  In particular, some items likely to change soon
+include:
+
+* The `image` section will get completely removed.
+
+* The `hardwareProfile` is currently exposed as a way to allow specifying
+  different hardware parameters for deployment.  By default, we will deploy
+  RHCOS to the first disk, but that may not be appropriate for all hardware.
+  The `hardwareProfile` is the field we have available to change that.  This
+  interface is subject to change.  In the meantime, hardware profiles can be
+  found here:
+  https://github.com/metal3-io/baremetal-operator/blob/master/pkg/hardware/profile.go#L48
+
+```yaml
+apiVersion: v1beta4
+baseDomain: test.metalkube.org
+metadata:
+  name: ostest
+compute:
+- name: worker
+  replicas: 1
+controlPlane:
+  name: master
+  replicas: 3
+  platform:
+    baremetal: {}
+platform:
+  baremetal:
+    apiVIP: 192.168.111.5
+    hosts:
+      - name: openshift-master-0
+        role: master
+        bmc:
+          address: ipmi://192.168.111.1:6230
+          username: admin
+          password: password
+        bootMACAddress: 00:11:07:4e:f6:68
+        hardwareProfile: default
+      - name: openshift-master-1
+        role: master
+        bmc:
+          address: ipmi://192.168.111.1:6231
+          username: admin
+          password: password
+        bootMACAddress: 00:11:07:4e:f6:6c
+        hardwareProfile: default
+      - name: openshift-master-2
+        role: master
+        bmc:
+          address: ipmi://192.168.111.1:6232
+          username: admin
+          password: password
+        bootMACAddress: 00:11:07:4e:f6:70
+        hardwareProfile: default
+    - name: openshift-worker-0
+        role: master
```

Comment:
Is worker-0 a master?

Reply:
copy/paste error

Reply:
are we going to fix it?

Reply:
https://github.com/openshift-metal3/kni-installer/pull/166

This PR needs rebasing pretty quick due to conflicts as things are merged, so would prefer to bring this over in a follow-up rather than kick CI from scratch again. I think there might be a few other doc fixes we'll need to bring over.